ro_help/hub/forms.py

```diff
@@ -16,6 +16,40 @@ class Meta:
         widgets = {"email": EmailInput()}
 
 
+class NGOForm(forms.ModelForm):
+    class Meta:
+        model = models.NGO
+        fields = ("__all__")
+
+    def clean(self):
+        cleaned_data = super().clean()
+        accepts_mobilpay = cleaned_data.get("accepts_mobilpay")
+        accepts_transfer = cleaned_data.get("accepts_transfer")
+        donations_description = cleaned_data.get("donations_description")
+        mobilpay_icc = cleaned_data.get("mobilpay_icc")
+        mobilpay_public_key = cleaned_data.get("mobilpay_public_key")
+        mobilpay_private_key = cleaned_data.get("mobilpay_private_key")
```

Comment:
Maybe is late, but instead of saving those in a file, we can use https://django-fernet-fields.readthedocs.io/en/latest/.

Reply:
We'll take care of this later